### DISCUSSION TITLE
See version

### DISCUSSION THREAD

**Discussion by hyperknot** [plausible/analytics#3107] [Self-Hosted Support] [answered]
Hi, is there a way to see the current running version in self-hosted mode? The docker compose says 1.5, but I'd love to know if it.s 1.5.0 or 1.5.1 or later.

I looked everywhere but I couldn't see the version in the UI. I even looked at docker logs, but version isn't shown. Why is it hidden so much?

**Comment by ruslandoga** [comment]
👋 @hyperknot 

Maybe something like this would work:

```console
$ cd hosting
$ docker compose exec plausible 'echo $BUILD_METADATA'
```

**Reply by ruslandoga** [reply]
I can't try it out right now as I'm on mobile internet and my instance doesn't seem to accept ipv6 connections.

**Reply by hyperknot** [reply]
```
OCI runtime exec failed: exec failed: unable to start container process: exec: "echo $BUILD_METADATA": executable file not found in $PATH: unknown
```

But it's not so critical for me, it'd be just nice to see it at the UI and in docker logs.

**Reply by ruslandoga** [reply, marked as answer]
What about

```console
$ docker compose exec plausible ash
# echo $BUILD_METADATA
```

**Reply by hyperknot** [reply]
Thanks, that works!

**Reply by c-heine** [reply]
Apart from that, it would still be nice to see a version number somewhere in the dashboard...
A little notice above the "Solely funded by our subscribers." would easily be enough.

**Reply by hyperknot** [reply]
Yes, please show it somewhere!
